{"name": "risk user -- @1.4 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/all_users_data/xlsx", "status": "passed", "start": 1610701821398, "stop": 1610701821953}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1610701821953, "stop": 1610701822109}], "parameters": [{"name": "url", "value": "/reconciliation/all_users_data/xlsx"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1610701821394, "stop": 1610701822109, "uuid": "bda8fd1c-408c-428d-8ac1-cb8cc313fe35", "historyId": "ad308c72d421c1e6a04fe8711c9cc284", "fullName": "features/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}